controllers/comments.controller.js

```diff
@@ -0,0 +1,17 @@
+const { selectArticle } = require('../models/articles.modles');
+const { selectCommentsForArticle } = require('../models/comments.models');
+
+exports.getCommentsForArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const promises = [
+    selectCommentsForArticle(article_id),
+    selectArticle(article_id),
+  ];
+  return Promise.all(promises)
+    .then((comments) => {
```

Comment:
These are more like resolved promises rather than 'comments', and the comments are only comments[0].